backend/app/domain/services/base/context.py

```diff
@@ -16,3 +20,40 @@ def increment_counter_total_samples_and_update_date(self, context_id: int) -> No
 
     def get_real_round_id(self, context_id: int) -> int:
         return self.context_repository.get_real_round_id(context_id)
+
+    def get_contexts(
+        self, tid: int, rid: int, method: str = "least_used", n: int = 1, tags=None
+    ):
+        """Get a context for annotation
```

Comment:
maybe it is a better idea to put the full variable names (e.x tid = task_id)) with that change, we can also remove the unnecessary information in Docs.